docs-md/web/utilities.md

```diff
@@ -0,0 +1,84 @@
+---
+title: JavaScript Utilities
+description: Capacitor's JavaScript Utilities
+url: /docs/web/utilities
+contributors:
+  - dotNetkow
+---
+
+# JavaScript Utilities
+
+<p class="intro">Capacitor has several JavaScript utilities useful for ensuring apps run successfully across multiple platforms with the same codebase. Simply import Capacitor then call the desired utility function:</p>
```

Comment:
"simply", "basically", etc. are frustrating words if developers are struggling to get something working

```suggestion
<p class="intro">Capacitor has several JavaScript utilities useful for ensuring apps run successfully across multiple platforms with the same codebase. To use them, import Capacitor and call the desired utility function:</p>
```